### PR TITLE
tcl: make tclConfig.sh executable

### PR DESCRIPTION
**Pull request by curiousgay**
This is needed for tcltls which I'm about to package.

### CODE REVIEW

File: srcpkgs/tcl/template [hunk 1]
```diff
@@ -26,6 +26,7 @@ post_extract() {
 do_install() {
 	make INSTALL_ROOT=${DESTDIR} install install-private-headers
 	ln -s tclsh8.6 ${DESTDIR}/usr/bin/tclsh
+	chmod +x ${DESTDIR}/usr/lib/tclConfig.sh
```
Comment (ericonr) [inline srcpkgs/tcl/template:29]:
```suggestion
	chmod 755 ${DESTDIR}/usr/lib/tclConfig.sh
```

Comment (curiousgay) [reply srcpkgs/tcl/template]: done

**Comment by ericonr** [comment]
I feel like `tcltls` is doing something wrong, if it needs that file to be executable... It just sets some variables, so really all it should do is be sourced by other scripts.

**Comment by curiousgay** [comment]
alright, there is only one line that needs to be patched to source that file

**Pull request closed without merging**